docs/SUMMARY.md

```diff
@@ -20,8 +20,8 @@
 * [Creating Application](user-guide/creating-application/README.md)
   * [Git Repository](user-guide/creating-application/git-material.md)
   * [Docker Build Configuration](user-guide/creating-application/docker-build-configuration.md)
-  * [Deployment Template](user-guide/creating-application/deployment-template.md)
-    * [Rollout](user-guide/creating-application/deployment-template.md)
+  * [Deployment Template](user-guide/creating-application/deployment-template/README.md)
+    * [Rollout](user-guide/creating-application/deployment-template/deployment-template.md)
```

Comment:
It should be
[Rollout Deployment](user-guide/creating-application/deployment-template/rollout-deployment.md)

Reply:
changed